basket/news/tests/test_views.py

```diff
@@ -161,7 +172,7 @@ def test_empty_email_invalid(self, update_user_mock):
         resp = views.subscribe(req)
         resp_data = json.loads(resp.content)
         self.assertEqual(resp_data["status"], "error")
-        self.assertEqual(resp_data["code"], errors.BASKET_INVALID_EMAIL)
+        self.assertEqual(resp_data["code"], errors.BASKET_USAGE_ERROR)
```

Comment:
Slight change here since we're now checking for email or token and if either has a false-ish value we return the usage error. Previously if email was at least specified in the data but empty it would return invalid email. I consider this an acceptable change since email (or token) is required and an empty email is basically no email.